app/controllers/computacenter/api/base_controller.rb

```diff
@@ -12,9 +12,17 @@ class Computacenter::API::BaseController < ApplicationController
 
 private
 
+  def identify_user!
+    @user = APIAuthenticationService.identify_user(bearer_token)
```

Comment:
you can safely inline this now and get rid of the service

```suggestion
    @user = APIToken.active.find_by(token: bearer_token)&.user
```

Reply:
Yep, agreed. Related, I've put a Trello card in to review the service objects we have - I feel like the concept has got muddled and they're becoming a dumping ground for stuff that in a previous life I might have put in the dreaded 'utils' library....